bowtie/report/src/data/report.js

```diff
@@ -0,0 +1,107 @@
+class Summary {
+  constructor(lines) {
+    // console.log((implementations))
+
+    this.lines = lines;
+    const runInfoData = this.lines[0];
+    const implementationsArray = Object.values(runInfoData.implementations);
+    // console.log(implementationsArray)
+
+    this.implementations = implementationsArray.sort((each, other) => {
+      const key = `${each.language}${each.name}`;
+      const otherKey = `${other.language}${other.name}`;
+      return key < otherKey ? -1 : 1;
+    });
+    this.did_fail_fast = false;
+    this.counts = {};
+    // console.log(this.counts)
+    this.initializeCounts();
+  }
+  initializeCounts() {
+    for (const each of this.implementations) {
+      this.counts[each.image] = new Count();
+    }
+  }
+
+  get total_cases() {
```

Comment:
Is there a reason for using a snake case? Generally, I would consider camel case to be an accepted standard for JS

Reply:
Not actually!😅 I'll change that.